### PR TITLE
15 lecture9 tutorials

### PR DESCRIPTION
**Pull request by seungwooj**
#15 

Using softmax and CrossEntropyLoss function to train MNIST data.
Issue: solve the TypeError in lec9_softmax_classifier_2.py

### CODE REVIEW

File: lec9_softmax_classifier_2.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+# from torch.autograd import Variable
+
+# Training settings
+batch_size = 64
+transform = transforms.Compose([
+    transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))
+])
+
+# MNIST Data Sets
+train_dataset = datasets.MNIST(root='../data', train=True, download=True, transform=transform)
```
Comment (hyoungseok) [inline lec9_softmax_classifier_2.py]: ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

File: lec9_softmax_classifier_2.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+test_dataset = datasets.MNIST(root='../data', train=False, transform=transform)
+
```
Comment (hyoungseok) [inline lec9_softmax_classifier_2.py]: too much space

File: lec9_softmax_classifier_2.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    for batch_idx, (data, target) in enumerate(train_loader):
+        # data, target = Variable(data), Variable(target)
+        optimizer.zero_grad()
+        output = model(data)  # TypeError 발생의 원인으로 생각되는 부분
+        loss = criterion(output, target)
+        loss.backward()
+        optimizer.step()
+
+        if batch_idx % 100 == 0:
+            print('Train Epoch: {} [{}/{} ({:0f}%)]\tLoss: {:.6f}'.format(
```
Comment (hyoungseok) [inline lec9_softmax_classifier_2.py]: 명시적 코딩 & 줄바꿈을 좀 더 가독성 좋게?

File: lec9_softmax_classifier_2.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        # data, target = Variable(data, volatile=True), Variable(target)
+        output = model(data)  # TypeError 발생의 원인으로 생각되는 부분
+        # sum up batch loss
+        test_loss += criterion(output, target).data.item()
+        # get the index of the max Log-probability
+        pred = torch.max(output.data, 1)[1]
+        correct += pred.eq(target.data.view_as(pred)).cpu().sum()
+
+    test_loss /= len(test_loader.dataset)
+    print('\nTest set: Average loss: {:.4f}, Accuracy: {}/{} ({:.0f}%)\n'.
```
Comment (hyoungseok) [inline lec9_softmax_classifier_2.py]: Same with above

File: lec9_softmax_classifier_2.py (hunk 1 above)
Comment (seungwooj) [reply lec9_softmax_classifier_2.py]: transform으로 수정하였습니다. 주의하겠습니다.

File: lec9_softmax_classifier_2.py (hunk 2 above)
Comment (seungwooj) [reply lec9_softmax_classifier_2.py]: erased one line.
